kotlinlib/src/mill/kotlinlib/KotlinModule.scala

```diff
@@ -153,6 +155,23 @@ trait KotlinModule extends JavaModule { outer =>
       )
   }
 
+  /**
+   * Compiler Plugin dependencies.
+   */
+  def kotlinCompilerPluginIvyDeps: T[Seq[Dep]] = Task { Seq.empty[Dep] }
+
+  /**
+   * The resolved plugin jars
+   */
+  def kotlinCompilerPluginJars: T[Seq[PathRef]] = Task {
+    val jars = defaultResolver().resolveDeps(
+      kotlinCompilerPluginIvyDeps()
+        // Don't resolve transitive jars
+        .map(d => d.exclude("*" -> "*"))
+    )
+    jars.toSeq
+  }
+
```

Comment:
I'd rather use the shorter `kotlinc` prefix instead of `kotlinCompiler`. We use both prefixes in the modules though.